core/src/main/java/com/linecorp/armeria/server/docs/ServiceSpecification.java

```diff
@@ -149,6 +152,11 @@ public ServiceSpecification(Iterable<ServiceInfo> services,
         this.exampleHeaders = ImmutableList.copyOf(requireNonNull(exampleHeaders, "exampleHeaders"));
     }
 
+    public ServiceSpecification withServiceRoute(Route serviceRoute) {
+        this.serviceRoute = serviceRoute;
```

Comment:
What about `EXACT`?

Reply:
I am adding that as default too, seems like it shouldn't hurt.

Reply:
We can't serve docserivce using `EXACT` so I think only one of `PREFIX`, `REGEX` and `GLOB` can be used.
If `REGEX` and `GLOB` are used, I thought we could just don't set it because we can't handle it. 😉

Reply:
Makes sense, even if we can handle it seems like not worth it. Why would somebody mount `DocService` under a glob or regex path? 😅 

About exact, this is related to how client code is generated, right? Because we are not serving a single static page but rather a page under path `/docs/#/...`?

Reply:
> Makes sense, even if we can handle it seems like not worth it. Why would somebody mount DocService under a glob or regex path?

That is correct. 😄 

>  Because we are not serving a single static page but rather a page under path /docs/#/...?

That is also correct. The client sends additional requests whose path is:
```
/docs/main.js?somehash
/docs/specification.json
/docs/versions.json
/docs/injected.js
/docs/schemas.json
/docs/assets/favicon.png
```

Reply:
@minwoox Please take a second look when you are free 😄 

Also, what is the standard procedure for testing those changes?

I am thinking about running `Annotated`, `GraphQL`, `Grpc` doc services and test everything is fine.

Reply:
We don't have automated tests for UI. 😓
We definitely need to set up front-end tests.